### PR TITLE
Register old SLE clients #16464 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Find channel names to register old SLE ciients.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

https://github.com/SUSE/spacewalk/issues/16464

### CODE REVIEW

**Comment by keichwa** [comment]
Hi Oscar @srbarrios , is this what we need in master (4.3).  If yes, I'll prepare backports for 4.2 and 4.1.  I hope this will the issue you reported.

**Review by 0rnela — APPROVED** [review]

**Comment by keichwa** [comment]
@mcalmer can you please check whether this PR is good enough?

File: modules/client-configuration/pages/clients-sle.adoc [hunk 1]
```diff
@@ -78,6 +78,18 @@ The channels you need for this procedure are:
...
+[NOTE]
+====
+To find channel names of older products, at the command prompt on the {productname} Server, as root, use the [command]``mgr-sync`` command:
+
+----
+mgr-sync list --help
+----
+
+Then specify the argument you are interested in.
+For example, [literal]``-c channels``.
```
Comment (mcalmer) [inline modules/client-configuration/pages/clients-sle.adoc:90]: `-c` is at the wrong place (funny - it works anyway). But we write better:

```
mgr-sync list channels [-c]
```
The option comes after the subcommand. "-c" is here "compact".

**Review by mcalmer — CHANGES_REQUESTED** [review]
please check my comment

File: modules/client-configuration/pages/clients-sle.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/clients-sle.adoc]: ok, thanks `;-)`

**Review by mcalmer — APPROVED** [review]

**Pull request merged**